app/lib/app/SecvisogramPage/View.js

```diff
@@ -320,6 +324,8 @@ function View({
   }
 
   const onNewHandler = () => {
+    uniqueProductId(true)
+    uniqueGroupId(true)
```

Comment:
This seems to be the wrong place: if I select "Cancel" in die dialogue the IDs are resetted... This should not be the case.

Reply:
Indeed, I changed it and extended the test.

Reply:
Thanks for extending the test. Please also extend the one for GroupIds

Reply:
Does it make sense to split those tests into separate testcases? I guess it makes it cleaner...